src/web/Storage.ts

```diff
@@ -14,7 +14,7 @@ import RX = require('../common/Interfaces');
 export class Storage extends RX.Storage {
     getItem(key: string): SyncTasks.Promise<string|undefined> {
         const value = window.localStorage.getItem(key);
-        return SyncTasks.Resolved<string|undefined>(value);
+        return SyncTasks.Resolved<string|undefined>(value || undefined);
```

Comment:
Why do you want to turn empty strings into undefined here? I don't think that's the right semantic.